ambry-tools/src/main/java/com.github.ambry/account/ServiceIdAccountGenTool.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.account;
+
+import com.github.ambry.tools.util.ToolUtils;
+import com.github.ambry.utils.Utils;
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import joptsimple.ArgumentAcceptingOptionSpec;
+import joptsimple.NonOptionArgumentSpec;
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
+import org.json.JSONArray;
+import org.json.JSONException;
+
+
+/**
+ * A tool to generate accounts for existing service IDs. Each account will share its name with the provided service ID
+ * and contain two containers, a default public and a default private container. These containers are meant to mirror
+ * the existing concept of private and public blobs and will be assigned accordingly by
+ * {@link com.github.ambry.frontend.AmbryBlobStorageService} when making POST requests with the service ID based API.
+ */
+public class ServiceIdAccountGenTool {
+
+  /**
+   * Entry point for the tool.
+   * @param args the command line arguments.
+   */
+  @SuppressWarnings("unchecked")
+  public static void main(String[] args) {
+    try {
+      OptionParser parser = new OptionParser();
+
+      ArgumentAcceptingOptionSpec<String> accountJsonFilePathOpt =
```

Comment:
maybe we should use a properties file like the rest of our tools

Reply:
I would prefer not in this case, since there are only 3 arguments for this tool.